models/migrations/v220.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	packages_model "code.gitea.io/gitea/models/packages"
+	container_module "code.gitea.io/gitea/modules/packages/container"
+
+	"xorm.io/xorm"
+	"xorm.io/xorm/schemas"
+)
+
+func addContainerRepositoryProperty(x *xorm.Engine) error {
+	switch x.Dialect().URI().DBType {
+	case schemas.SQLITE:
+	case schemas.ORACLE:
```

Comment:
`ORACLE` is not supported yet.

Reply:
Other migrations use `ORACLE` too (110, 210).

Reply:
Yeah, just realize them. But I think we should also delete them. What do you think about that?

Reply:
Sure, I don't need it. 😄